the-collab-lab/tcl-45-smart-shopping-list
# Readme edit

## Description

This PR updates the `README.md` with a brief and concise summary of the app. I have also added team members and mentors with their respective contact information. 

## Related Issue

Closes #69 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Testing Steps / QA Criteria

1. Open README.md and open preview to the side to see the layout of the README file.
2. Read through information to see if there is more/less to add, and whether the flow of information feels clear.
